app/models/manageiq/providers/amazon/inventory/parser/storage_manager/ebs.rb

```diff
@@ -76,14 +76,11 @@ def link_volume_to_disk(volume_hash, attachments)
 
       dev = File.basename(a['device'])
 
-      disk_hash = {
```

Comment:
ah this is related to the targeted refresh, this code doesn't work for targeted without the find_or_build_by